src/main/java/model/data/value/TemporalValue.java

```diff
@@ -13,6 +13,13 @@
 
 	private String format;
 
+	/**
+	 * Return a null instance.
+	 */
+	TemporalValue() {
+		this(null);
```

Comment:
this(null) betekend roep de constuctor aan met argument null. Dus er staat dan TemporalValue(null); en de constructor handelt de null value af